### PR TITLE
Update to knapsack's readme.md 

### PR DESCRIPTION
**Pull request by SaiSakethAluru**

### CODE REVIEW

**Comment by SaiSakethAluru** [comment]
PR #629 Updated README.md describing the knapsack algorithm

File: code/mathematical-algorithms/count_trailing_zeroes/Count_trailing_zeroes_factorial.cpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#include <iostream>
```
Comment (AdiChat) [inline code/mathematical-algorithms/count_trailing_zeroes/Count_trailing_zeroes_factorial.cpp]: Kindly remove this file 👍

File: code/mathematical-algorithms/count_trailing_zeroes/count_trailing_zeros_factorial.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+def findTrailingZeroes(n):
```
Comment (AdiChat) [inline code/mathematical-algorithms/count_trailing_zeroes/count_trailing_zeros_factorial.py]: Kindly remove this file 👍

**Comment by SaiSakethAluru** [comment]
Removed the files.

**Comment by AdiChat** [comment]
Looks good 👍 

Hope you are enjoying your journey with **OpenGenus**

**Pull request merged**